OrbitGl/EventTrack.cpp

```diff
@@ -16,6 +16,10 @@ EventTrack::EventTrack(TimeGraph* a_TimeGraph) : Track(a_TimeGraph) {
   m_Color = Color(0, 255, 0, 255);
 }
 
+std::string EventTrack::GetTooltip() const {
+  return "Left-click and drag to select samples.";
```

Comment:
Nit: I think that it's a bit cleaner to not have the "." in the tooltip. (Single data point: Chrome does not have "." in their tooltips at the end of the sentence.)

Reply:
I was undecided about this, so this tipped the scales and I removed the "." everywhere ;-)